tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -100,6 +100,11 @@ export class RunsDataTable {
     return this.data.some((row) => row['selected']);
   }
 
+  handleSelectAll(event: MouseEvent) {
+    event.preventDefault();
+    this.onAllSelectionToggle.emit(this.data.map((row) => row.id));
```

Comment:
Can you delete the function getRunIds()? It now seems to be unused.

Reply:
Good catch. Removed.